File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    const divide = s.length/2
    if(Number.isInteger(divide))
        //정수 ㅡ> 짝수
         {
             answer = s[divide-1]+s[divide]
        
     } else {
         answer = s[Math.trunc(divide)]
     }
    return answer;
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        if(s.length()%2==0){
            int quo = s.length()/2;
            answer+= s.substring(quo-1,quo+1);
        } else{
            int quo = s.length()/2;
            answer+= s.substring(quo,quo+1);
        }
        return answer;
    }
}